default/widgets.py

```diff
@@ -0,0 +1,6 @@
+from django.forms.widgets import Widget
+
+
+class Display(Widget):
+    def render(self, name, value=None, attrs=None):
+        return '<span>{}</span>'.format(value)
```

Comment:
I guess it's possible to make it a bit more friendly with some CSS? Is the current implementation working for autocomplete?